dot/digest/README.md

```diff
@@ -0,0 +1,95 @@
+with the [`NewHandler`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#NewHandler) function and started
+along with the Gossamer node's other services. The digest `Handler` maintains two `goroutines`, `handleBlockImport` and
+`handleBlockFinalisation`, which handle GRANDPA changes that occur when blocks are imported and finalised,
+respectively - these `goroutines` listen for messages on channels provided by the
+[`BlockState`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/state#BlockState).
+
+The digest `Handler` also exposes two public functions:
+[`NextGrandpaAuthorityChange`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#Handler.NextGrandpaAuthorityChange)
+and [`HandleDigests`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#Handler.HandleDigests).
+`NextGrandpaAuthorityChange` is consumed by the
+[GRANDPA service](https://pkg.go.dev/github.com/ChainSafe/gossamer/lib/grandpa#Service) and is used during the pre-vote
+and pre-commit [equivocation stages](https://medium.com/polkadot-network/polkadot-consensus-part-2-grandpa-fb1963ef6c70)
+to determine the block number of the next GRANDPA authority change.
+
+`HandleDigests` is invoked by the core service for each new block; this function reads the consensus digests from the
+block and dispatches actions based on the messages the digests contain. The digest `Handler` supports digests from the
+BABE (authorship) and GRANDPA (finalisation) consensus engines - the following sections describe the messages in these
+digests and the actions Gossamer takes when it receives them.
+
+## BABE Messages
+
+[BABE](https://wiki.polkadot.network/docs/learn-consensus#block-production-babe) is a proof-of-stake block authorship
+algorithm that randomly assigns block authorship rights over periods of time referred to as "epochs". One of the
+benefits of BABE is that it is able to provide a blockchain runtime with secure, decentralized randomness. Furthermore,
+BABE seeks to ensure the liveness of a blockchain by defining multiple tiers of potential block producers for each
+epoch. BABE digests may contain 1 of 3 messages, each of which are described below.
+
+### Next Epoch
+
+This message is issued by the runtime on the first block of every epoch - it provides the BABE consensus engine with the
+authority set and randomness for the _next_ epoch.
+
+### Disabled
+
+A message of this type will contain the ID of an authority; this authority should cease all authority functionality and
+all other authorities should ignore any authority-related messages from the identified authority.
+
+### Next Config
+
+Messages of this type may only be issued in the first block of an epoch. These types of messages supply configuration
+parameters that should be applied from the _next_ epoch onwards. The parameters in this configuration relate to how
+backup authorities are selected.
```

Comment:
```suggestion
parameters that should be applied from the _next_ epoch onwards. The parameters in this configuration are `C`, which is used to determine the BABE threshold, and a `SecondarySlots` boolean, which is set to true if secondary slot block production is enabled.
```

Reply:
this is kinda hard cause we don't want to describe the BABE threshold and secondary slots here (or do we?)

Reply:
This feels like too much detail for me. I'm worried that it will create more questions than it answers. For instance, what is "the BABE threshold"?

Reply:
That's a rhetorical question BTW 😅 I know (kind of 😅) what it means, but I'm worried that it won't be transparent to everyone reading this.